lta/rest_server.py

```diff
@@ -87,6 +87,26 @@ def initialize(self, db: Any, check_claims: CheckClaims, *args: Any, **kwargs: A
         self.db = db
         self.check_claims = check_claims
 
+class FilesActionsBulkCreateHandler(BaseLTAHandler):
+    @lta_auth(roles=['system'])
+    async def post(self) -> None:
+        req = json_decode(self.request.body)
+        if 'files' not in req:
+            raise tornado.web.HTTPError(400, reason="missing files field")
+        if not isinstance(req['files'], list):
+            raise tornado.web.HTTPError(400, reason="files field is not a list")
+        if not req['files']:
+            raise tornado.web.HTTPError(400, reason="files field is empty")
```

Comment:
Recommend consolidating these `if` statements into one.  We shouldn't hand-hold the user at the expense of extra lines of code.  A single reason is sufficient.

Reply:
I have two lines of thinking on this, it depends on if we consider an empty list to be a valid input or not:

1. If an empty list is an error, it's nice to be explicit as to why the request was rejected.

2. If an empty list is not an error, then I'd think to remove Line 98-99. The logic would add 0 files to the database and return 201 with an empty array of UUIDs. The user requested we `bulk_create` nothing, so that's what we do.

Reply:
I'm inclined to say an empty list is an error, because you can't create nothing (so you shouldn't have called in the first place).  I guess to break the tie, what does @vbrik think?

Reply:
I'm going to leave the error on empty list because it prompts the error case. I'm not sure that it's right to rely on that behavior, but it is what it is for this PR.